src/traversal/printRelayOSSQuery.js

```diff
@@ -28,11 +26,11 @@ type PrinterState = {
   fragmentNameByText: {[fragmentText: string]: string};
   fragmentTexts: Array<string>;
   variableCount: number;
-  variableMap: {[variableID: string]: Variable};
+  variableMap: Map<mixed, Variable>;
```

Comment:
I went ahead and use the existing Flow type, `Variable`, here. I don't particularly feel like the new contents of this `Variable` object (`type` and `variableID`) are properly described by the name Variable, but I'm not sure if changing this to something like `VariableDetails` or something is necessary. I just left it as-is. I've also never used Flow, so if this is flat out incorrect, I can change it. :)